modules/aws/rds_serverless_cluster/main.tf

```diff
@@ -112,3 +111,5 @@ resource "random_password" "password" {
 resource "random_id" "secrets_id" {
   byte_length = 8
 }
+
```

Comment:
avoid blank newlines. I would suggest we use an editorconfig

Reply:
Is that VSCode thingy? (Jetbrains Fiend)

Could do a prehook to TF fmt, as that's relatively development platform agnostic. (Seperate issue though with that one)

Reply:
[editorconfig](https://editorconfig.org/) is an IDE-agnostic way of defining coding styles for a project. However, it seems enforcing a _single_ trailing newline isn't something it covers yet: https://github.com/editorconfig/editorconfig/issues/269

A text file should end with **exactly one** newline - you can [enforce a newline at the end of a file in your editor](https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file), but terraform fmt is not opinionated about vertical whitespace.

Reply:
I've switched it on in Webstorm 
![image](https://github.com/answerdigital/terraform-modules/assets/106234256/1b9c2139-b03e-4f4e-afaa-b0fb8422ea26)

Don't know about other IDE's

Reply:
Dont ask why I use webstorm for this.
It's a mystery lost to time

Reply:
Make sure you tick that that one too - all files should end with a line break